htdocs/accountancy/admin/account.php

```diff
@@ -272,6 +272,7 @@
 	if ($search_pcgsubtype) $param .= '&search_pcgsubtype='.urlencode($search_pcgsubtype);
     if ($optioncss != '') $param .= '&optioncss='.$optioncss;
 
+    /*
```

Comment:
If you remove this, when you make a search on a field in the list, you will trigger the load of the chart of account.

Reply:
@eldy 
ok i see, i split the form and remove javascript